test/integration_test/s3/getObject.spec.js

```diff
@@ -1,23 +1,29 @@
-var helpers = require('../helpers');
-var AWS = helpers.AWS;
-var baseName = helpers.bucketBaseName;
+var expect = require('chai').expect;
+var baseName = 'aws-sdk-js-integration';
+var AWS = require('../../../lib/aws');
 describe('download integrity', function() {
   var params;
   var s3;
-  function putObject(customParams, done) {
-    params = {
-      Bucket: customParams.Bucket || params.Bucket,
-      Key: customParams.Key || params.Key,
-    };
+  function createBucket(customParams, done) {
+    params.Bucket = customParams.Bucket || params.Bucket;
     s3.createBucket({Bucket: params.Bucket}).promise().then(function() {
       return s3.waitFor('bucketExists', {Bucket: params.Bucket}).promise()
     }).then(function() {
-      return s3.putObject({
-        Bucket: params.Bucket,
-        Key: params.Key,
-        Body: customParams.Body || params.Body
-      }).promise();
-    }).then(function() {
+      if (typeof done === 'function') done();
+    }).catch(function(err) {
+      throw new Error('Cannot create bucket: ' + err);
+      process.exit(1);
```

Comment:
Does process.exit(1) ever get called? The error that's thrown right above it should cause you to break out of this catch block.